### PR TITLE
 Fixed: In the 'backupSuccess' step of a TSQLDatabaseBackupEvent event handler, you couldn't delete the target database file

### PR DESCRIPTION
**Pull request by edwinyzh**
Note: I needed to delete the target db file after after I've compress the db file into a .zip file

### CODE REVIEW

**Comment by edwinyzh** [comment]
Hi ab, would you merge this fix? Thanks.

File: SynSQLite3.pas [hunk 1]
```diff
@@ -3290,6 +3291,7 @@   TSQLDatabaseBackupThread = class(TThread)
     property DestDB: TSQLDatabase read fDestDB;
     /// the raised exception in case of backupFailure notification
     property FailureError: Exception read fError;
+    property BackupDestFile: RawUTF8 read fBackupDestFile;
```
Comment (synopse) [inline SynSQLite3.pas]: please add a /// documentation comment

File: SynSQLite3.pas [hunk 2]
```diff
@@ -5776,12 +5779,17 @@ procedure TSQLDatabaseBackupThread.Execute;
           raise ESQLite3Exception.Create('Backup process forced to terminate');
         SleepHiRes(fStepSleepMS);
       until false;
+
```
Comment (synopse) [inline SynSQLite3.pas]: please don't add void lines to the source ;)

File: SynSQLite3.pas (hunk 1 above)
Comment (edwinyzh) [reply SynSQLite3.pas]: comment added

**Pull request merged**